kml/modules/location_ground.py

```diff
@@ -0,0 +1,52 @@
+"""
+Class LocationGround to use instead of tuple for coordinates.
```

Comment:
Remove class name (since it is redundant and might change (makes it so that there is one less thing to change)).